test/index.test.ts

```diff
@@ -366,3 +366,12 @@ test('throw error in beforeAll, afterAll, beforeEach, afterEach', async () => {
   expect(bench.getTask('AE test')!.result!.error).toBe(AEerror);
   expect(bench.getTask('AA test')!.result!.error).toBe(AAerror);
 });
+
+test('remove non-existent task', () => {
```

Comment:
try to make the description more descriptive.

```suggestion
test('removing non-existing task should not throw', () => {
```